defaults/main.yml

```diff
@@ -4,6 +4,8 @@ prometheus_version: 2.2.1
 prometheus_config_dir: /etc/prometheus
 prometheus_db_dir: /var/lib/prometheus
 
+prometheus_service_hardening: True
```

Comment:
I think we should disable this by default, as the older operating systems are still quite popular.